udp/discover.go

```diff
@@ -23,18 +23,18 @@ type MulticastOption interface {
 }
 
 // Discover sends GET to multicast address and wait for responses until context timeouts or server shutdown.
```

Comment:
multicast or unicast right ? I would suggest describe it more in docs comment

Reply:
Fixed